repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -13,12 +13,15 @@ class PyIcon4py(PythonPackage):
     components for weather and climate models."""
 
     url = "git@github.com:C2SM/icon4py.git"
+    git = 'ssh://git@github.com/C2SM/icon4py.git'
 
     homepage = "https://github.com/C2SM/icon4py"
 
     maintainers = ['samkellerhals']
 
-    version('main', branch='main', git='ssh://git@github.com/C2SM/icon4py.git')
+    version('main', branch='main', git=git)
+    version('0.0.1', tag='v0.0.1', git=git)
```

Comment:
If we don't use it I suggest to delete it

Reply:
Deleting it for now. New tags will follow in a week or so.